README.md

```diff
@@ -1 +1,11 @@
-# Group-Assignment2-Cinema-Project
\ No newline at end of file
+# Group-Assignment2-Cinema-Project
+
+Best Practices
```

Comment:
Add "#" to make this a heading :)

Reply:
as a new point in the list add:
- Prettier for formating in vscode with single quotation instead of double.